ios/Podfile

```
@@ -0,0 +1,64 @@
+require File.join(File.dirname(`node --print "require.resolve('expo/package.json')"`), "scripts/autolinking")
+require File.join(File.dirname(`node --print "require.resolve('react-native/package.json')"`), "scripts/react_native_pods")
+
+require 'json'
+podfile_properties = JSON.parse(File.read(File.join(__dir__, 'Podfile.properties.json'))) rescue {}
```

RuboCop (1.76.1)

[convention] Avoid using `rescue` in its modifier form.

(Style/RescueModifier)